core/chains/evm/forwarders/forwarder_manager_test.go

```diff
@@ -41,7 +42,8 @@ var SimpleOracleCallABI = evmtypes.MustGetABI(operator_wrapper.OperatorABI).Meth
 
 func TestFwdMgr_MaybeForwardTransaction(t *testing.T) {
 	lggr := logger.Test(t)
-	db := pgtest.NewSqlxDB(t)
+	testChainId := testutils.FixtureChainID
```

Comment:
BTW you can use `github.com/smartcontractkit/chainlink/v2/core/chains/evm/testutils` to get FixtureChainID which is not coupled with Core.